mp/src/materialsystem/stdshaders/pbr_ps30.fxc

```diff
@@ -33,16 +35,14 @@ const float3 cAmbientCube[6]				: register( PSREG_AMBIENT_CUBE );
 const float4 g_EyePos						: register( PSREG_EYEPOS_SPEC_EXPONENT );
 const float4 g_FogParams					: register( PSREG_FOG_PARAMS );
 const float4 g_FlashlightAttenuationFactors	: register( PSREG_FLASHLIGHT_ATTENUATION );			// On non-flashlight pass
-//const float4 g_RimBoost						: register( PSREG_FLASHLIGHT_POSITION_RIM_BOOST );
-//const float4 g_MetalnessRoughness			: register( PSREG_FLASHLIGHT_SCREEN_SCALE );
 const float4 g_FlashlightPos				: register( PSREG_SPEC_RIM_PARAMS );
 const float4x4 g_FlashlightWorldToTexture	: register( PSREG_FLASHLIGHT_TO_WORLD_TEXTURE );
 PixelShaderLightInfo cLightInfo[3]			: register( PSREG_LIGHT_INFO_ARRAY );				// 2 registers each - 6 registers total (4th light spread across w's)
-//const float4 g_LightmapCoords[3]			: register( PSREG_AMBIENT_CUBE );
 const float4 g_BaseColor					: register( PSREG_SELFILLUMTINT );
 
-#define PI								3.141592
-#define EPSILON							0.00001
+static const float PI = 3.141592;
+static const float EPSILON = 0.00001;
+static const float ENVMAPLOD = 6.0;
```

Comment:
Could you add a comment explaining what the 6.0 stands for? (1 roughness == 4x4 texture, 0 roughness == 256x256 texture)

Reply:
// 6 is the lowest cubemap mipmap we need. in a 256x256 cubemap, it gives us 4x4